main.py

```diff
@@ -456,18 +470,22 @@ def get_data_from_github(url: str, properties: Dict[str, str]) -> Project:
                 releases = []
             else:
                 raise e
-        if len(releases) > 300:
-            logger.warning(
-                "To many tags ({}), skipping for performance reasons.".format(
-                    len(releases)
-                )
-            )
-            releases = []
+
         invalid_version_strings = []
         extracted = [
             analyse_tag(release, project_info, invalid_version_strings)
             for release in releases
         ]
+        extracted = [v for v in extracted if v is not None]
+        extracted.sort(key=lambda x: LooseVersion(re.sub(r"[^0-9.]", "", x.version)))
+        if len(extracted) > 300:
+            logger.warning(
+                "To many tags ({}), limiting to 300 for performance reasons.".format(
+                    len(extracted)
+                )
+            )
+            extracted = extracted[-100:]
```

Comment:
That's a limit of 100 (I'm fine with both 100 and 300; It should only be consistent)